pkgs/development/compilers/llvm/4/clang/default.nix

```diff
@@ -13,20 +13,32 @@ let
       mv clang-tools-extra-* $sourceRoot/tools/extra
     '';
 
-    buildInputs = [ cmake libedit libxml2 llvm python ];
+    buildInputs = [ cmake libedit libxml2 llvm python python.pkgs.sphinx ];
```

Comment:
since this is a rebuild for darwin anyway, can you move cmake, ... to `nativeBuildInputs`